src/trackPlayer.ts

```diff
@@ -115,22 +108,22 @@ async function removeUpcomingTracks(): Promise<void> {
 /**
  * Skips to a track in the queue.
  */
-async function skip(trackIndex: number, initialPosition?: number): Promise<void> {
-  return TrackPlayer.skip(trackIndex, optionalNumberToDefault(initialPosition))
+async function skip(trackIndex: number, initialPosition = 0): Promise<void> {
```

Comment:
> this might break things.. the default used to be -1 before and now its 0 -- it might have unexpected consequences.
Looked into this, and indeed in Android a seek is done when initialPosition is 0 and in iOs only when initialPosition > 0. (Which could be a bug in itself)

I will revert this